packages/headless/src/features/insight-user-actions/insight-user-actions-state.ts

```diff
@@ -2,8 +2,9 @@ import {InsightAPIErrorStatusResponse} from '../../api/service/insight/insight-a
 
 export interface UserActionTimeline {
   precedingSessions: UserSession[];
-  session: UserSession;
+  session: UserSession | undefined;
   followingSessions: UserSession[];
+  caseSessionFound: boolean;
 }
```

Comment:
You no longer need the `caseSessionFound` since now we always will have a `session` if we have a ticket creation date.
```suggestion
export interface UserActionTimeline {
  precedingSessions: UserSession[];
  session: UserSession | undefined;
  followingSessions: UserSession[];
  caseSessionFound: boolean;
}
```